src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -71,12 +64,6 @@
     <DefineConstants>$(DefineConstants);FEATURE_REFLECTION_GETASSEMBLIES;FEATURE_REFLECTIONONLYLOAD;FEATURE_EVENT_ARGS_MUST_EXTEND_EVENTARGS</DefineConstants>
   </PropertyGroup>
 
-  <!-- Include PDB in package -->
-
-  <ItemGroup>
-    <None Include="$(OutputPath)/**/$(AssemblyName).pdb" Pack="true" PackagePath="lib" Visible="false" />
-  </ItemGroup>
-
```

Comment:
No longer necessary thanks to `AllowedOutputExtensionsInPackageBuildOutputFolder`. We can't do the same in `FakeItEasy.Extensions.ValueTask`, though, because it would include the PDB for `netstandard2.1`, and I couldn't find a way to exclude it.

If you prefer, I can move that to a separate PR. I did it in this one, because the `AllowedOutputExtensionsInPackageBuildOutputFolder` approach is mentioned in the SourceLink instructions.

Reply:
I am content as is. Thanks for the explanation.